fourmat/lint.py

```diff
@@ -17,6 +17,72 @@
 SNAPSHOT_REGEX = r".*\/snapshots\/snap_.*\.py"
 CONFIGURATION_FILES = (".flake8", "pyproject.toml")
 
+
+class PyProjectConfig:
+    """
+    Helper methods to access pyproject.toml settings
+    """
+
+    PATH = Path("pyproject.toml")
+    CONF = None
+
+    @property
+    def _data(self):
+        if self.CONF is None:
+            try:
+                import toml
+                self.CONF = toml.load(self.PATH)
+            except ImportError:
+                self.CONF = {}
```

Comment:
not sure how/why this would happen, as i am pretty sure deps like black and isort depend on it, and I'd rather specify it.